proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+
+<lynX> This seems to be asking for an exchange standard, not a legislation. Standard document formats however exist, also means for upload — so if the industry hasn't established a standard practice in this field maybe that's because they have no economic incentive to do so? A legal policy could make it mandatory for publishing platforms to accept a uniform simple document upload format. Not clear to me which political problem this really addresses.
+
+# 2 
+# 28.06.2018 Input DSC Vienna1 - 18-06-28 
+
+First of all, the participants determined that they do not have a profound knowledge on the topic. The topic literally is very technical and interdisciplinary as well due to its interconnection and impacts to other political fields, but we nevertheless tried our best ;-). We ask for your understanding that we are contributing to the first block of questions democratising technologies only.
+One member stated that the fact that older citizens cannot use modern technologies due to physical reasons is to be considered as highly undemocratic (e.g. reading on a smartphone). More tax-generated public investments shall be undertaken in that field (research).
```

Comment:
We could require that all smartphone apps must also be usable from any computer. Technically this is already possible, but hardly anyone has an Android emulator on their laptop. - lynx